src/components/SingleArticle.jsx

```diff
@@ -5,25 +5,27 @@ import styles from "../css/SingleArticle.module.css";
 import CommentsCard from "./CommentsCard";
 import Vote from "./Vote";
 
-function SingleArticle() {
-  const [singleArticle, setSingleArticle] = useState({});
-  const [isLoading, setIsLoading] = useState(true)
 
+function SingleArticle({ username }) {
+  const [singleArticle, setSingleArticle] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
   const { article_id } = useParams();
 
-  useEffect(() => {
-    fetchArticle(article_id).then(({ articles }) => {
-      setSingleArticle(articles[0]);
-      setIsLoading(false)
-    });
-  }, [isLoading],[singleArticle]);
-
+  useEffect(
+    () => {
+      fetchArticle(article_id).then(({ articles }) => {
+        setSingleArticle(articles[0]);
+        setIsLoading(false);
+      });
+    },
+    [isLoading],
```

Comment:
isLoading doesn't need to be in a dependency array here. Please take another look at your loading pattern. In addition, useEffect can only take one dependancy array, so even if it were to be included this would be incorrect syntax